src/main/java/com/irtimaled/bbor/common/interop/CommonInterop.java

```diff
@@ -59,4 +65,11 @@ public static void playerLoggedOut(ServerPlayerEntity player) {
     public static void playerSubscribed(ServerPlayerEntity player) {
         EventBus.publish(new PlayerSubscribed(player.getId(), new ServerPlayer(player)));
     }
+
+    public static <T extends AbstractBoundingBox> AbstractRenderer<T> registerRenderer(Class<? extends T> type, Supplier<AbstractRenderer<T>> renderer) {
```

Comment:
This should be in ClientInterop - CommonInterop contains code that is used in the server-only versions like Bukkit